src/test/java/com/wikia/webdriver/testcases/adstests/TestNoAdsLoggedInUsers.java

```diff
@@ -35,11 +35,9 @@ public TestNoAdsLoggedInUsers(String wikiName, String path) {
   }
 
   private void loginSteps() {
```

Comment:
Why is it call this way? :japanese_ogre: I can't see any "steps" there ;) Wouldn't it be easier to call it just `login()`? :)